js/mexc.js

```diff
@@ -39,6 +40,11 @@ module.exports = class mexc extends Exchange {
                 'fetchDeposits': true,
                 'fetchFundingRate': true,
                 'fetchFundingRateHistory': true,
+                'fetchFundingRates': false,
+                'fetchIndexOHLCV': false,
+                'fetchIsolatedPositions': false,
+                'fetchLeverage': false,
```

Comment:
You sure there's no api endpoint that returns the max leverage on a market?

Reply:
There is an endpoint for that but I wasn't sure if that's what fetchLeverage was meant for because the only example I've seen of fetchLeverage doesn't return the max leverage on the market

Reply:
Don't worry about the `fetchLeverage` method right now, it looks like we only have it on okex, and we don't create a unified response for it, just remove it from has

Reply:
Okay will do